src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -35,6 +35,8 @@ namespace OpenTelemetry.Instrumentation.AspNetCore.Implementation
     internal class HttpInListener : ListenerHandler
     {
         internal const string ActivityOperationName = "Microsoft.AspNetCore.Hosting.HttpRequestIn";
+        internal static readonly bool IsNet7OrGreater = typeof(HttpRequest).Assembly.GetName().Version.Major >= 7;
+        internal static readonly string FrameworkActivitySourceName = "Microsoft.AspNetCore";
```

Comment:
add link to where this source name is coming from.

Reply:
Might wanna make this a permalink. Linked from main the line number could change. You can highlight a line in github and then click the 3 little dots when hovering over the line to select copy permalink.